src/opnsense/www/js/widgets/Gateways.js

```diff
@@ -48,12 +48,12 @@ export default class Gateways extends BaseTableWidget {
     }
 
     async _fetchGateways() {
-        const data = await this.ajaxCall('/api/routes/gateway/status');
-        if (!data.items || !data.items.length) {
+        const data = await this.ajaxCall('/api/routing/settings/searchGateway/');
```

Comment:
the `status` symbol is now undefined, since the API changed we probably need to recheck the keys

Reply:
I have fixed the status.

Reply:
trailing slash? besides it made the lint-acl unhappy:

```
Registered widget endpoints do not match:
<<<<<<< src/opnsense/www/js/widgets/Gateways.js
/api/routing/settings/searchGateway/
========
/api/routes/gateway/status
>>>>>>> src/opnsense/www/js/widgets/Metadata/Core.xml
```

Reply:
https://github.com/opnsense/core/commit/23ec00f641b300347124b89c27873975db029f20